index.html

```diff
@@ -357,25 +357,19 @@ <h2>
             </div>
     
             
-            <form>
+            <form action="https://formspree.io/f/xnqrgeew" method="POST"> 
+                <input type="text" id="name" maxlength="30" name="name" placeholder="Yeramis" required>
+                
+                <input type="email" id="email" name="email" placeholder="natajaynj14@gmail.com" required >
+                
+                <textarea id="textarea" maxlength="500" placeholder="write your message here" required></textarea>
```

Comment:
- [ ] I think it would be better if you made the inputs with more height and decrease the width as shown in the figma since it is very important to stick with the figma as much as possible 💯🚀🔥

## I will show you the difference 🔥🚀
- Your design
![image](https://user-images.githubusercontent.com/58553711/199229835-1864ec1e-0eb3-4dfb-8dd6-b3599557cbac.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/199229937-c69b54a5-1e2f-4d5e-9624-d7d4a97d2fce.png)